imports/ui/pages/panel/jobs/index.js

```diff
@@ -31,3 +31,9 @@ Template.jobs.helpers({
     return Jobs.find({ nome: eval("/.*"+searchValue+".*/i"), categoria: eval("/.*"+findByValue+".*/i")}, {sort: sortBy} ).fetch();
   }
 });
+
+Template.jobs.onRendered(function(){
+  $(document).ready(function(){
```

Comment:
I THINK this ` $(document).ready(function(){` is not necessary, because the `Template.jobs.onRendered(function(){` already waits the page to load